docs/CONTRIBUTING.md

```diff
@@ -26,15 +26,49 @@ Alternative methods if you don't like Github (also fully supported):
 It is likely however once a patch is ready to apply a maintainer
 will push it to a github PR, and merge via Homu.
 
-Commit message style
---------------------
+## [Commit message style](#commit-message-style)
 
 Please look at `git log` and match the commit log style, which is very
 similar to the
 [Linux kernel](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git).
 
 You may use `Signed-off-by`, but we're not requiring it.
 
+**General Commit Message Guidelines**:  
+
+1. Title
+    - Specify the context or category of the changes .e.g `lib` for library changes, `docs` for document changes, `bin/<command-name>` for command changes, etc.
+    - Begin the title with the first letter of the first word capitalized.
+    - Aim for less than 50 characters, otherwise 72 characters max.
+    - Do not end the title with a period.
+    - Use an [imperative tone](https://en.wikipedia.org/wiki/Imperative_mood).
+2. Body
+    - Separate the body with a blank line after the title.
+    - Begin a paragraph with the first letter of the first word capitalized.
+    - Each paragraph should have be within 72 characters.
+    - Content should be about what was changed and why this change was made.
+    - `Closes:` and `Aprroved by:` are autogenerated by `rh-atomic-bot`
```

Comment:
That's more or less an implementation detail. Maybe let's drop this here and in the template below?
One should still do `Closes: #N` if the commit fixes an issue.

Reply:
Addressed this: https://github.com/ostreedev/ostree/pull/1694/commits/51a51264d4c049628435a8afdaa73d5fb536ea34. How does this look?